build/configure.js

```diff
@@ -157,9 +164,14 @@ function buildRules(grunt, options, commons, callback) {
 				// Translate rules
 				metadata.rules[rule.id] = parseMetaData(rule, 'rules');
 			}
-			descriptions.push([rule.id, entities.encode(rule.metadata.description), rule.tags.join(', '), rule.enabled === false ? false : true]);
+			descriptions.push([
```

Comment:
inconsistent - if you're going to put `{` on a new line, why not `[`? Although I prefer no new line....

Reply:
I think it treats Arrays and Objects differently.